backend/server.js

```diff
@@ -37,12 +36,12 @@ const io = require("socket.io")(server, {
   pingTimeout: 60000,
   cors: {
     origin: ["http://localhost:3000", "https://chat-app-two-theta-52.vercel.app", "https://chatapi.gitleet.live"],
-    // credentials: true,
```

Comment:
Why you removed this?

Reply:
Wait I'll fix it

Reply:
It was anyways commented. So I removed it. But to handle a few functions, I will uncomment it and push it again